app/src/main/jni/def.h

```diff
@@ -0,0 +1,19 @@
+#include <android/log.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#ifndef PICTUREN_DEF_H
+#define PICTUREN_DEF_H
+
+#ifdef __ANDROID__
```

Comment:
if you're using checks for \_\_ANDROID\_\_, then I believe you should put `#include <android/log.h>` line inside ifdef.

From the other hand, we do not expect this code to be built without NDK, so there is no need to have it(ifdef) at all. 
(But we would need to have it is case when we want the code to be compiled without NDK, i.e. for some other platforms)